packages/@aws-cdk/aws-eks/lib/helm-release.ts

```diff
@@ -0,0 +1,86 @@
+import cfn = require('@aws-cdk/aws-cloudformation');
+import { Construct, Stack } from '@aws-cdk/core';
+import { Cluster } from './cluster';
+
+/**
+ * Helm Release Properties.
+ */
+export interface HelmReleaseProps {
+  /**
+   * The EKS cluster to apply this configuration to.
+   *
+   * [disable-awslint:ref-via-interface]
+   */
+  readonly cluster: Cluster;
+
+  /**
+   * The name of the release.
+   */
+  readonly name: string;
+
+  /**
+   * The name of the chart.
+   */
+  readonly chart: string;
+
+  /**
+   * The chart version to install.
+   * @default If this is not specified, the latest version is installed
```

Comment:
Use the value `-` for default in case you only want to describe the default behavior:

```
@default - If this is not specified.....
```